docs/onboarding_toolkit.md

```diff
@@ -1 +1,172 @@
-#  Onboarding toolkit
+# Ansible Project Onboarding
+
+Welcome to the Ansible ecosystem of projects! This document guides you 
+through the steps to take to become part of the Ansible ecosystem.
+
+# Why be part of the Ansible ecosystem
+Being a part of the Ansible ecosystem means you are tapping into a large and
+thriving community of passionate people working and volunteering in Ansible.
+
+The major benefits of becoming part of the Ansible ecosystem include:
+
+* A presence on docs.ansible.com/ecosystem for your project documentation,
+ as well as being part of the official Ansible ReadTheDocs space.
+* A presence on the (future) Ansible.com to introduce your project.
+* Dedicated communications channels (chat and forum) to communicate with your
+community and attract other community members to your project.
+
+
+
+# Setting up your GitHub repository
+
+The Ansible ecosystem projects exist in the following two GitHub repositories:
+
+* ansible - Access is controlled by Red Hat
+* ansible-community - Access is controlled by the community, 
+under Red Hat sponsorship
+
+We recommend most Ansible projects use the ansible-community GitHub repository.
+Repository transfers are available if your project already exists somewhere else in GitHub.
```

Comment:
How to request a repo transfer one? (if we do a "how to" section for new projects, consider existing ones).